src/cms/api/page/models/page.js

```diff
@@ -5,4 +5,15 @@
  * to customize this model
  */
 
-module.exports = {};
+const slugValidator = require("../../../config/functions/slugValidator.js");
```

Comment:
Slugs for pages should always begin with a forward slash.  The regex allows me to enter a slug without the forward slash.

Is there something we can do to make it less brittle?  for example, chaining adding a couple of replace() statements to gatsby-node.js will ensure there is always a beginning and ending slash even if the user didn't enter one....
```
      actions.createPage({
        path: page.Slug.replace(/^\/|^/, `/`).replace(/\/$|$/, `/`),
        component: require.resolve(`./src/templates/${page.Template}.js`),
        context: { page },
      })
```